tpm2/src/getkey/main.c

```diff
@@ -31,6 +31,8 @@ static void usage(const char *prog)
           "  -i or --input         Path to file containing the kmyth-sealed client's certificate private key.\n"
           "  -l or --client        Path to file containing the client's certificate.\n\n"
           "Server Information --\n"
+          "  -t or --type          The type of the key server. Valid values include 'kmip' and 'simple'.\n"
```

Comment:
The README.md file in the tpm2 directory contains usage information. We should also update that as part of this pull request to keep it current.

Reply:
Good point. I'll update that right now.